MovieApp/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,299 @@
+        let layout = UICollectionViewFlowLayout()
+        layout.itemSize = CGSize(width: screenWidth*0.26, height: 40)
+        layout.scrollDirection = .vertical
+        layout.sectionInset = UIEdgeInsets(top: 0, left: screenWidth*0.06, bottom: 0, right: screenWidth*0.06)
+        layout.minimumInteritemSpacing = 10
+        crewView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+ 
+        
+    }
+    
+    private func customizeViews() {
+        
+        movieImageView.load(url: URL(string: movieDetails.imageUrl)!)
+        movieImageView.contentMode = .scaleAspectFill
+        movieImageView.clipsToBounds = true
+        view.addSubview(movieImageView)
+        
+        userScoreLabel.attributedText = NSMutableAttributedString().bold(String(format: "%.1f", arguments: [movieDetails.rating]), fontSize: 20).normal(" User Score", fontSize: 16)
+        userScoreLabel.textColor = .white
+        view.addSubview(userScoreLabel)
+        
+        movieTitleLabel.attributedText = NSMutableAttributedString().bold(movieDetails.name, fontSize: 30).normal(String(format: " (%d)", arguments: [movieDetails.year]), fontSize: 30)
+        movieTitleLabel.textColor = .white
+        movieTitleLabel.adjustsFontSizeToFitWidth = true
+        view.addSubview(movieTitleLabel)
+        
+        releaseDateLabel.attributedText = NSMutableAttributedString().normal(movieDetails.releaseDate, fontSize: 15)
+        releaseDateLabel.textColor = .white
+        view.addSubview(releaseDateLabel)
+        
+        genreLabel.attributedText = NSMutableAttributedString().normal(categoriesToString(categories: movieDetails.categories), fontSize: 15).bold(lengthIntToString(length: movieDetails.duration), fontSize: 15)
+        genreLabel.textColor = .white
+        view.addSubview(genreLabel)
+        
+        favoriteButton.backgroundColor = .gray
+        favoriteButton.layer.cornerRadius = 17.5
+        let homeSymbolConfiguration = UIImage.SymbolConfiguration(pointSize: 17, weight: .regular)
+        let homeImage = UIImage(systemName: "star", withConfiguration: homeSymbolConfiguration)
+        favoriteButton.setImage(homeImage, for: .normal)
+        view.addSubview(favoriteButton)
+        
+        overviewLabel.attributedText = NSMutableAttributedString().bold("Overview", fontSize: 30)
+        overviewLabel.textColor = .black
+        view.addSubview(overviewLabel)
+        
+        summaryLabel.attributedText = NSMutableAttributedString().normal(movieDetails.summary, fontSize: 18)
+        summaryLabel.textColor = .black
+        summaryLabel.lineBreakMode = .byWordWrapping
+        summaryLabel.numberOfLines = 0
+        view.addSubview(summaryLabel)
+        
+        //crewView.translatesAutoresizingMaskIntoConstraints = false
+        crewView.delegate = self
+        crewView.dataSource = self
+        for crew in movieDetails.crewMembers {
+            crewView.register(crewViewCell.self, forCellWithReuseIdentifier: crew.name)
```

Comment:
I don't think this is necessary. You should register one cell type only once. So, you should replace lines 106-108 with `crewView.register(crewViewCell.self, forCellWithReuseIdentifier: "CrewViewCell")`